Classes/URKArchive.mm

```diff
@@ -315,8 +351,44 @@ + (BOOL)urlIsARAR:(NSURL *)fileURL
     return [NSArray arrayWithArray:fileInfos];
 }
 
+- (nullable NSString *)firstVolumePath:(NSString *)filePath
```

Comment:
The idea of the overloads (same to previous comment) was to make this methods useful for checking a file that is not the one loaded during the init process. For example, while searching for the first volume and checking if it is in fact a volume. I'm gonna remove them, so It'll be always checking for `self.fileURL`.